sdk/transaction.go

```diff
@@ -9,22 +9,18 @@ import (
 	"context"
 	"fmt"
 	"net/http"
-
-	"github.com/proximax-storage/go-xpx-utils/net"
 )
 
 type TransactionService struct {
 	*service
 	BlockchainService *BlockchainService
 }
 
-// returns Transaction for passed transaction id or hash
-func (txs *TransactionService) GetTransaction(ctx context.Context, id string) (Transaction, error) {
+// returns Transaction for passed transaction group and hash
+func (txs *TransactionService) getTransaction(ctx context.Context, group string, id string) (Transaction, error) {
 	var b bytes.Buffer
 
-	url := net.NewUrl(fmt.Sprintf(transactionRoute, id))
-
-	resp, err := txs.client.doNewRequest(ctx, http.MethodGet, url.Encode(), nil, &b)
+	resp, err := txs.client.doNewRequest(ctx, http.MethodGet, fmt.Sprintf("%s/%s/%s", transactionsRoute, group, id), nil, &b)
```

Comment:
Instead "%s/%s/%s" you can create template const string for that=D

Reply:
Which template did you mean? This [one](https://golang.org/pkg/text/template/#pkg-overview) or?

Reply:
Like this)